django_project/frontend/templates/emails/invitation_email.html

```diff
@@ -0,0 +1,212 @@
+</td></tr></tbody></table></div>
+<div class="pc2-8169 ogf g" style="font-size:0;text-align:left;
+direction:ltr;display:inline-block;vertical-align:top;width:2.8169%;">
+<table border="0" cellpadding="0" cellspacing="0" role="none" width="100%">
+    <tbody><tr><td style="vertical-align:top;padding:0;">
+<table border="0" cellpadding="0" cellspacing="0" role="none" 
+style width="100%">
+<tbody></tbody></table>
+</td></tr></tbody></table></div>
+<div class="pc56-6901 ogf c" style="font-size:0;text-align:left;
+direction:ltr;display:inline-block;vertical-align:middle;width:56.6901%;">
+<table border="0" cellpadding="0" cellspacing="0" role="none" 
+style="border:none;vertical-align:middle;" width="100%"><tbody>
+<tr><td margin-right="right" class="x" style="font-size:0;word-break:break-word;">
+<div style="text-align:right;"><p style="Margin:0;text-align:right;
+mso-line-height-alt:125%"><span style="font-size:16px;
+font-family:Inter,Arial,sans-serif;font-weight:700;color:#ffffff;
+line-height:125%;">South African Wildlife Population System</span>
+</p></div></td></tr></tbody></table></div></div></td></tr></tbody></table>
+</div>
+<div class="r  pr-16 pl-16" style="background:#fffffe;
+background-color:#fffffe;margin:0px auto;max-width:600px;">
+<table align="center" border="0" cellpadding="0" cellspacing="0" 
+role="none"style="background:#fffffe;background-color:#fffffe;width:100%;">
+<tbody><tr><td style="border:none;direction:ltr;font-size:0;
+padding:32px 32px 32px 32px;text-align:left;">
+<div class="xc536 ogf c" style="font-size:0;
+text-align:left;direction:ltr;display:inline-block;
+vertical-align:middle;width:100%;">
+<table border="0" cellpadding="0" cellspacing="0" role="none" 
+style="border:none;vertical-align:middle;" width="100%"><tbody>
+<tr><td align="left" class="x" style="font-size:0;word-break:break-word;">
+<div style="text-align:left;">
+<p style="Margin:0;text-align:left;mso-line-height-alt:150%">
+<span style="font-size:12px;font-family:Inter,Arial,sans-serif;
+font-weight:400;color:#000000;line-height:150%;">Dear User,
+</span></p>
+<p style="Margin:0;mso-line-height-alt:150%">
+<span style="font-size:12px;font-family:Inter,Arial,sans-serif;
+font-weight:400;color:#000000;line-height:150%;">&nbsp;</span></p>
+<p style="Margin:0;mso-line-height-alt:150%"><span style="font-size:12px;
+font-family:Inter,Arial,sans-serif;font-weight:400;color:#000000;
+line-height:150%;">You have been invited as a {{ role }} of {{ organisation }} on the South African Wildlife Population System (SAWPS)
+</span></p>
+<p style="Margin:0;mso-line-height-alt:150%"><span 
+style="font-size:12px;font-family:Inter,Arial,sans-serif;
+font-weight:400;color:#000000;line-height:150%;">&nbsp;</span></p>
+<p style="Margin:0;mso-line-height-alt:150%"><span 
+style="font-size:12px;font-family:Inter,Arial,sans-serif;
+font-weight:400;color:#000000;line-height:150%;">To get started, please click on the following button to Register
```

Comment:
@DONOTFORGET-BYTE I think the email can be sent to either non-existing user or existing user. If we send to existing user, it seems better to use different email rather than invite the user again to Register. Can we differentiate the email for non-existing user invite and existing user invite?

Reply:
Okay I'm adding this now , when the user is existing they get the invite which will send them to the site and register them only to the organisation and not on the platform since they already exist

Reply:
I have resolved this